src/field_theory/galois.lean

```diff
@@ -393,6 +393,27 @@ begin
   exact (linear_equiv.findim_eq (intermediate_field.lift2_alg_equiv K⟮x⟯).to_linear_equiv).symm,
 end
 
+theorem tfae [finite_dimensional F E] :
```

Comment:
The code looks good, but this is one of those theorems that I think deserve a nice docstring. It's one of those results that appears in all textbook accounts of Galois theory.
For the same reason, it makes sense to mention this result in the module docstring of this file.

Reply:
I've added some docstrings